schemas/JSON/manifests/v1.2.0/manifest.installer.1.2.0.json

```diff
@@ -385,6 +411,19 @@
       },
       "description": "List of OS architectures the installer does not support"
     },
+    "UnsupportedArguments": {
+      "type": [ "array", "null" ],
+      "uniqueItems": true,
+      "items": {
+        "type": "string",
+        "title": "UnsupportedArgument",
+        "enum": [
+          "log",
+          "location"

```

Comment:
Regarding #910 - Would it be appropriate to include `silent` in this enum?

Reply:
Some arguments in the Silent field are required in the schema, so I don't know if that makes sense. Manifests that don't support `--silent` shouldn't exist (hopefully).